plugins/ipam/dhcp/daemon.go

```diff
@@ -37,6 +37,7 @@ const resendCount = 3
 
 var errNoMoreTries = errors.New("no more tries")
 
+// DHCP represents a DHCP server that can allocate IP address leases
```

Comment:
This is not quite true. Better:
The DHCP daemon is a DHCP client daemon which is controlled and used by the DHCP IPAM plugin